### DISCUSSION TITLE
Low Mach number assumption

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#12842] [Q&A]
Dear developer,

In FDS algorithm, low Mach number assumption is adopted to support the low-speed flow like the most common fire. It decomposes the pressure $p$ into a background pressure plus a perturbation. So, in the overall procedure, the background pressure is computed while ignoring the perturbation? 

What confuses me is why we make that assumption and FDS can not be used for rapid discharge of pressure vessels for that assumption.

Could you give some detailed explanation or some information to understand it well?

Have a nice day.

LI

**Comment by drjfloyd** [comment]
Both are computed in the overall procedue:
-The perturbation pressure is used in the transport equations to drive flow via H (see 2.5 in the Tech Guide for the definition of H). 
-Change in background pressure (used for the equation of state) is computed per 3.3.6 in the Tech Guide.  

FDS was not developed with the intent of being a general purpose CFD code valid for all possible applications. Most of the fires occurring in the built environment are low speed flows. The low Mach number assumption improves the speed of the solver as FDS does not need to take timesteps capable of tracking pressure waves.

**Reply by lihy213** [reply]
That is to say, one of the significances of adopting this assumption is to improve the speed of the solver, and another to satisfy the particular algorithms of FDS for solving transport equations and some quantities. Right?
